neo_batterylevelshutdown/neo_batterylevelshutdown.py

```diff
@@ -52,15 +59,13 @@ def setup_gpio_pin(pinNum, direction):
 def blink_LEDxTimes(pinNum, times):
     """Blink the LED a certain number of times"""
     try:
-        with open("/sys/class/gpio/gpio{}/value".format(pinNum), "w") as pin:
-            for _ in range(0, times):
-                pin.write(PIN_LOW)
-                time.sleep(LED_FLASH_DELAY_SEC)
+        for _ in range(0, times):
+            with open("/sys/class/gpio/gpio{}/value".format(pinNum), "w") as pin:
                 pin.write(PIN_HIGH)
-                time.sleep(LED_FLASH_DELAY_SEC)
-
-            # make sure that we turn the LED off
-            pin.write(PIN_HIGH)
+            time.sleep(LED_FLASH_DELAY_SEC)
+            with open("/sys/class/gpio/gpio{}/value".format(pinNum), "w") as pin:
```

Comment:
Is there a reason we open the same file twice per loop iteration? If so, please add a comment explaining why.